tests/compiler/test_docs.py

```diff
@@ -37,26 +37,6 @@ def test_doc_string_on_new_relation(snippetcompiler):
     assert types["__config__::File"].get_attribute("host").comment.strip() == "Each file needs to be associated with a host"
 
 
-def test_doc_string_on_relation(snippetcompiler):
```

Comment:
Isn't this test still relevant with the new syntax? Or do we have another test for it already?

Reply:
Yes. We should keep it.